test_conformance/basic/test_enqueue_map.cpp

```diff
@@ -211,10 +215,10 @@ int test_enqueue_map_image(cl_device_id deviceID, cl_context context, cl_command
 
     for( size_t q = 0; q < imageSize * imageSize * 4; q++ )
     {
-      if( initialData[ q ] != finalData[ q ] )
+      if( referenceData[ q ] != finalData[ q ] )
       {
         log_error( "ERROR: Sample %d (coord %d,%d) did not validate! Got %d, expected %d\n", (int)q, (int)( ( q / 4 ) % imageSize ), (int)( ( q / 4 ) / imageSize ),
-                                    (int)finalData[ q ], (int)initialData[ q ] );
+                                    (int)finalData[ q ], (int)hostPtrData[ q ] );
```

Comment:
Nit-pick, can we print from `referenceData` instead of `hostPtrData`? I think it reads a little better based on the previous condition using `referenceData` and `finalData` (also the memory is not the same).